trikot-viewmodels-declarative-flow/viewmodels-declarative-flow/src/commonMain/kotlin/com/mirego/trikot/viewmodels/declarative/properties/VMDRichText.kt

```diff
@@ -0,0 +1,7 @@
+package com.mirego.trikot.viewmodels.declarative.properties
+
+data class VMDRichTextSpan(val range: IntRange, val transform: VMDRichTextTransform)
+
+sealed class VMDRichTextTransform
```

Comment:
This could be an interface if there is no need for it to be a class

Reply:
Good idea. It can become a sealed class in the future if we need to extend it.